netlify/constants.ts

```diff
@@ -5,13 +5,20 @@ import {
   chainInfo,
 } from "@govtechsg/tradetrust-utils/constants/supportedChains";
 
-export const ALLOWED_ORIGINS = [
-  "http://127.0.0.1:3000",
-  "http://localhost:3000",
-  "https://creator.tradetrust.io",
-  "https://dev.tradetrust.io",
-  "https://tradetrust.io",
-];
+export const ALLOWED_ORIGINS =
+  process.env.NODE_ENV === "test"
```

Comment:
You see if you wanna make this `dev` or not for consistency with other repos. But ok too if stick with `test`.